pkg/users/account-logs-panel.jsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState, useEffect } from 'react';
+
+import { Card, CardTitle, CardBody, DataList, DataListItem, DataListItemRow, DataListItemCells, DataListCell, Text } from '@patternfly/react-core';
```

Comment:
Do you have some mockup for this? 
The issue is that I am afriad you 're using the wrong component here.
If you want that header lines (table header) to be present you can't use data list, you need to use table, or our implementation of it, aka ListingTable.


When using this the mobile mode is broken, as headers appear as a cell. 
![Screen Shot 2021-09-27 at 15 50 21](https://user-images.githubusercontent.com/14921356/134922031-dc7aebc9-b0dd-410a-8b14-89715b4782bf.png)

Reply:
> Do you have some mockup for this?

The mockup is here https://raw.githubusercontent.com/cockpit-project/cockpit-design/9a65b44ffa9cf60a6eaf02044e797d0046b77d6b/users/login-warning.png

> The issue is that I am afraid you 're using the wrong component here.
> If you want that header lines (table header) to be present you can't use data list, you need to use table, or our implementation of it, aka ListingTable.

Aha!

Reply:
It's a ListingTable with a contains-list class (we already use it) on the table body.